src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -122,12 +122,12 @@ class ProxyService(proxyConfig: ProxyConfig,
 
   private def proxyInternal(userInfo: UserInfo, googleProject: GoogleProject, clusterName: ClusterName, request: HttpRequest): Future[HttpResponse] = {
     logger.debug(s"Received proxy request for user user $userInfo")
-    clusterDateAccessedActor ! UpdateDateAccessed(clusterName, googleProject, Instant.now())
     getTargetHost(googleProject, clusterName) flatMap {
       case ClusterReady(targetHost) =>
         // If this is a WebSocket request (e.g. wss://leo:8080/...) then akka-http injects a
         // virtual UpgradeToWebSocket header which contains facilities to handle the WebSocket data.
         // The presence of this header distinguishes WebSocket from http requests.
+        clusterDateAccessedActor ! UpdateDateAccessed(clusterName, googleProject, Instant.now())
```

Comment:
Can I make a tiny nitpick... :)

The comment here refers to L131, so maybe put this line BEFORE the comment block to make that more clear.